lib/Benchmark/Remote/Launcher.php

```diff
@@ -131,6 +137,7 @@ private function resolvePhpBinary()
         }
 
         // otherwise try and find it in PATH etc.
+        /** @var string|null $phpBinary */
```

Comment:
Perhaps this should be amended upstream?

Reply:
Yeah, we could make the return type `?string` ?

Reply:
Hmm, just noticed it's `symfony/process`, so here we go: https://github.com/symfony/symfony/pull/29702.